### PR TITLE
Examples of suspend_chains

### PR DESCRIPTION
**Pull request by joseppinilla**
Just thought it would be useful to have.
But also, wanted to point out that, since the internal pins are also returned in the embedding, a naive use of the embedding with the drawing capabilities, will cause an error if show_labels=True is used.

Maybe there's nothing to do here but the user should be aware of this. Or maybe the drawing package should handle it... or... networkx shouldn't use label keys to address the pos dictionary, which is what causes the error.

### CODE REVIEW

**Review by arcondello — APPROVED** [review]

**Review by arcondello — CHANGES_REQUESTED** [review]
I like the example, but we should add a test to CI so that we know it doesn't get out of date.

**Comment by boothby** [comment]
> But also, wanted to point out that, since the internal pins are also returned in the embedding

Uhhh... can you rephrase that in the form of a bug report?  Because that's awful and we shouldn't be returning those.  I'll have time to fix that late next week

**Comment by boothby** [comment]
I've added these examples as tests in response to @arcondello's change request, and patched the bug in PR #88 .

**Comment by boothby** [comment]
I've submitted a pull request to @joseppinilla's repo removing the note about the bug resolved in #88.  @arcondello's changes are resolved and I'm happy with this example (provided that note is removed).

**Review by boothby — CHANGES_REQUESTED** [review]
Strike lines 28-30 (accepting my PR to @joseppinilla's repo)

**Review by boothby — APPROVED** [review]

**Review by arcondello — APPROVED** [review]
